scripts/vm/hypervisor/kvm/setup_agent.sh

```diff
@@ -224,5 +224,17 @@ then
     setenforce 0
 fi
 
+which aria2c
+if [ $? -gt 0 ]
+then
+    yum install epel-release -y
+    yum install aria2 -y
```

Comment:
We can add aria2 dependency on rpms, see the spec files.

Reply:
I also added it on cloud.spec, should I remove it from here then?